### PR TITLE
Update base.py for issue #182, #186

### PR DESCRIPTION
**Pull request by dadr**
Changed the way `sum` was computed in `getImagesTotal()` to avoid black screen with "no (new) images could be found" in the warning log.

### CODE REVIEW

**Comment by mrworf** [comment]
Let me know tomorrow if these changes worked as expected and we'll merge then

**Comment by dadr** [comment]
After a day, this is still running without a hitch.

**Comment by dadr** [comment]
I've added a commit to this PR - it now addresses #186 as well.

File: services/base.py [hunk 1]
```diff
@@ -694,7 +696,7 @@ def requestUrl(self, url, destination=None, params=None, data=None, usePost=Fals
           break
         except:
           logging.exception('Issues downloading')
-        time.sleep(tries / 10) # Back off 10, 20, ... depending on tries
+        time.sleep(tries * 10) # Back off 10, 20, ... depending on tries
```
Comment (mrworf) [inline services/base.py:699]: Oups

**Pull request merged**